.env.example

```
@@ -0,0 +1,4 @@
+DB_NAME=s2
+DB_USER=s2
+DB_PASS=s2
+API_URL=http:localhost:8000
```

Looks like this should be:
```suggestion
API_URL=http://localhost:8000
```